integration/framework_test.go

```diff
@@ -1,4 +1,5 @@
-//
+// +build core
+
```

Comment:
Since no test would work without this "core" code, let's not tag it
```suggestion
```

Reply:
I don't think this works like that, but I will try it.